src/js/components/ResultBody/ResultsBody.js

```diff
@@ -1,24 +1,94 @@
-import React from "react";
-import Movie from "../Movie/Movie";
+import React, {Component} from "react";
+import Movie from "../Movie";
 import "./ResultsBody.css"
+import {loadMoreMovies, loadMovies} from "../../store/fetchData/actions";
+import {connect} from "react-redux";
+import InfiniteScroll from "react-infinite-scroll-component";
+import {NoFilmsFound} from "../NoFilmsFound";
+import {Loader} from "../Loader";
 
-function ResultsBody() {
-
-    return (
-        <div className="album jumbotron mb-0">
-            <div className="container">
-                <div className="row">
-                    <Movie/>
-                    <Movie/>
-                    <Movie/>
-                    <Movie/>
-                    <Movie/>
-                    <Movie/>
-                    <Movie/>
-                </div>
+class ResultsBody extends Component {
+
+    constructor(props) {
+        super(props);
+        this.fetchNextMovies = this.fetchNextMovies.bind(this);
```

Comment:
it's good. But also you can write decorator `Autobind` or use existing from `npm`, and need include decorators in `babel`